### PR TITLE
GSoC 2017 | Added contribution links in the README and project report

### PR DESCRIPTION
**Pull request by Deepanshu2017**
This Pull Request contains the links that are added to the [README.md](https://github.com/MultithreadCorner/Hydra.Python/blob/master/README.md) and the [project report](https://github.com/MultithreadCorner/Hydra.Python/blob/master/docs/project_report.rst) which shows the documentation of the Hydra.Python and the list of contribution I made over the span of last 3 months of Google Summer of Code 2017.

### CODE REVIEW

**Comment by AAAlvesJr** [comment]
Provide a description to the PR please.

File: README.md [hunk 1]
```diff
@@ -1,10 +1,7 @@
...
-
 Hydra.Python
 ============
 
-Python 2.7+ and 3.x bindings for the [Hydra C++ library](https://github.com/MultithreadCorner/Hydra/).
-
+Python 2.7+ and 3.x bindings for the [Hydra C++ library](https://github.com/MultithreadCorner/Hydra/
 The bindings are produced with [pybind11](http://pybind11.readthedocs.io/). The project makes use of [CMAKE](https://cmake.org/).
 
 You can find the Hydra.Python [here](https://github.com/MultithreadCorner/Hydra.Python/tree/GSoC2017-release).
```
Comment (AAAlvesJr) [inline README.md]: Need improve the spelling here. 
You can find the Hydra.Python - > You can find  Hydra.Python GSoC release ...

**Comment by AAAlvesJr** [comment]
I have no more comments... Eduardo do you have some?

**Comment by Deepanshu2017** [comment]
Thank you for merging the pull request!

**Comment by eduardo-rodrigues** [comment]
If you give me just 4 minutes to comment that's kind of tough, no ;-)? BTW I was having lunch so could have taken more like 30 minutes ...
I see a little thing and will comment in a sec.

**Pull request merged**